test/unit/model/import/Factory/CsvTemplateSampleLineFactoryTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2021  (original work) Open Assessment Technologies SA;
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoQtiItem\test\unit\model\import\Factory;
+
+use oat\generis\test\TestCase;
+use oat\taoQtiItem\model\import\Factory\CsvTemplateSampleLineFactory;
+use oat\taoQtiItem\model\import\TemplateInterface;
+
+class CsvTemplateSampleLineFactoryTest extends TestCase
+{
+    public function setUp(): void
+    {
+        $this->subject = new CsvTemplateSampleLineFactory();        
+    }
+    public function testfindMetadataByClassUri(): void
+    {        
+        $template = $this->createMock(TemplateInterface::class);
+        $sampleLines = $this->subject->getSampleLines($template);
+        $this->assertNotNull($sampleLines);
+    }
```

Comment:
Lets add a blank line to separate contexts. It is easier to read and PSR-12 compliant.

```suggestion
    public function setUp(): void
    {
        $this->subject = new CsvTemplateSampleLineFactory();        
    }
    
    public function testfindMetadataByClassUri(): void
    {        
        $template = $this->createMock(TemplateInterface::class);
        $sampleLines = $this->subject->getSampleLines($template);
        
        $this->assertNotNull($sampleLines);
    }
```

Reply:
Done